src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/APISecurityConfigFactory.kt

```diff
@@ -12,9 +12,16 @@ import org.vaccineimpact.orderlyweb.models.PermissionRequirement
 import org.vaccineimpact.orderlyweb.security.authorization.OrderlyWebAPIAuthorizer
 import org.vaccineimpact.orderlyweb.security.clients.*
 
-class APISecurityConfigFactory(
-        private val requiredPermissions: Set<PermissionRequirement>
-) : ConfigFactory
+interface APISecurityConfigFactory : ConfigFactory
```

Comment:
Mocking this class was a bit of a nightmare, I think because static bits of it seemed to mean that real methods were called. So I just made an interface for it instead.